### PR TITLE
Modified the appearance of the guide

### PR DESCRIPTION
**Pull request by Ahmad-Ayyoub**

### CODE REVIEW

**Comment by yeekangc** [comment]
@evelinec, can you please review and work with @Ahmad-Ayyoub further on this as necessary?

For example, the name of the CDI guide may still need to be changed.

**Comment by evelinec** [comment]
I think @Ahmad-Ayyoub had the correct cdi guide url now. It'd be the one when the guide is published. About the title, we need to change when there's a final decision on the cdi guide title. 

Other changes seem good. I'll read through the guide again after merge.

**Comment by evelinec** [comment]
@Ahmad-Ayyoub Pls handle the issues raised in the last 3 commits. Then we'd merge.

File: README.adoc [hunk 1]
```diff
@@ -276,8 +276,9 @@ request to check the service states again. This time, the expected response code
 Lastly, the method asserts whether the state of the `inventory` service is `DOWN` and the state of the
 `system` service is `UP`.
 
-In addition, a few endpoint tests, `InventoryEndpointTest.java` and `SystemEndpointTest.java`, have been provided for you to
-test the basic functionality of the `inventory` and `system` services. If a test failure occurs, then you must have
+In addition, a few endpoint tests, `start/src/test/java/it/io/openliberty/guides/inventory/InventoryEndpointTest.java`
```
Comment (evelinec) [inline README.adoc]: Remove the "," in "a few endpoint tests,".

**Pull request merged**